test/oli_web/live/delivery/instructor_dashboard/overview/recommended_actions_tab_test.exs

```diff
@@ -280,4 +290,20 @@ defmodule OliWeb.Delivery.InstructorDashboard.Overview.RecommendedActionsTest do
              )
     end
   end
+
+  defp set_schedule_for_page(hours, section_page) do
+    end_date = DateTime.utc_now() |> DateTime.add(hours, :hour) |> DateTime.truncate(:second)
+
+    section_page
+    |> Ecto.Changeset.change(%{graded: true})
+    |> Repo.update()
+
+    Oli.Delivery.Sections.SectionResource
+    |> where([sr], sr.resource_id == ^section_page.resource.id)
```

Comment:
This query MUST also include the `section_id` in it, otherwise it is going to change the schedule of this resource in possibly another course section.